docs/reference/sql/endpoints/rest.asciidoc

```diff
@@ -118,7 +118,7 @@ Controls if the CSV header should be included in the response or not. It can tak
 * `delimiter` (default: `,`)
 +
 Indicates which character should be used to separate the CSV values. The following values cannot be used for this attribute:
-apostrophe (`"`), carriage-return (`\r`) and new-line (`\n`). The tab (`\t`) can also not be used, the `tsv` format needs to be used
+double quotes (`"`), carriage-return (`\r`) and new-line (`\n`). The tab (`\t`) can also not be used, the `tsv` format needs to be used
```

Comment:
```suggestion
double quote (`"`), carriage-return (`\r`) and new-line (`\n`). The tab (`\t`) can also not be used, the `tsv` format needs to be used
```
since all the rest are also singular.